### ISSUE TITLE
Bring back the ability to use keys shorter than 6 characters please

### ISSUE THREAD

**Issue by mobeeniqbal** [nvda-es/NVDARemote#46]
Hi.
I'm currently using TeleNVDA-2025.3.3 with NVDA 2024.4.2. With the latest update to the add-on, I have found that I am unable to type a key which is shorter than 6 characters whilst controlling a machine. Whilst I can understand the need to have long keys for security purposes, we have machines that have been assigned shorter keys over the years that it would be inconvenient for us to change. Can you please make the setting optional as opposed to a requirement so we're able to use shorter keys that are less than 6 characters in length? We're using a privately hosted NVDA remote server so security isn't a problem. I've managed to resolve this issue temporarily by editing dialogs.py but it would be nice if the value was configurable. Many thanks.

**Comment by jmdaweb** [comment]
Hi,
I understand your point of view and apologise for the inconveniences. Short keys are really helpful, specially on private environments where security is controlled in another way. Unfortunately, this change has been made in response to a CVE. Making it optional would increase the risk of the vulnerability again. Although it may require a great effort, I recommend taking some time to upgrade your environment with longer keys. During the migration, returning to the previous add-on version or editing dialogs.py are good alternatives.
Regards.
